packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/src/index.ts

```diff
@@ -127,6 +128,11 @@ server.use(
   })
 );
 
+/**
+ * The health check endpoint
+ */
+server.get('/api/healthz', healthCheckMiddleware());
```

Comment:
What is your opinion about using the Express Router to hide the "healthcheck" middleware API path? I think we shouldn't expect customer to change the endpoint path and it's not customizable.
We can expose it as a "healthcheck"

Reply:
yea, i think that is a good reason to introduce healthcheck router

Reply:
@yavorsk Yeah, I think we don't need to expose it from SDK as a "healthcheckRouter" since it's just for a single endpoint, probably "healthcheck" is enough. (e.g. cookierParser middleware, etc.)

Reply:
yep, makes sense. updated.